src/components/TransactionDetails/Transfer/Transfer.less

```diff
@@ -0,0 +1,55 @@
+@import "../../../views/resources/css/variables.less";
+.transaction-details-outer-container{
+    max-height: 2.8rem;
+    overflow-y: scroll;
+  }
+  .transaction-details-item-inner-container {
+    display              : grid;
+    grid-row-gap         : .1rem;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows   : 1fr 1fr 1fr;
+    grid-template-areas  : 'a b'
+      'c c'
+      'd d';
+    
+  
+    .transaction-row-outer-container:nth-child(1) {
+      grid-area: a;
+    }
+  
+    .transaction-row-outer-container:nth-child(2) {
+      grid-area: b;
+    }
+  
+    .transaction-row-outer-container:nth-child(3) {
+      grid-area: c;
+    }
+  
+    .transaction-row-outer-container:nth-child(4) {
+      grid-area: d;
+    }
```

Comment:
this is not acceptable as is. What happens if any more "areas" are added ?

Please do not used "named areas" like that. If you are required to use `display: grid`, then please use _percentages_ to fill the correct space (and it must add up to 100%, **always**.)